src/fparser/common/splitline.py

```diff
@@ -249,94 +250,58 @@ def splitquote(line, stopchar=None, lower=False, quotechars="\"'"):
     current closing quotation character to be specified.
```

Comment:
Add a comment that explains that a double-quote-character is an 'escaped quote'.

Reply:
Thanks!